src/osd/scrubber/scrub_backend.cc

```diff
@@ -311,6 +307,13 @@ void ScrubBackend::update_authoritative()
   }
 }
 
+#ifdef NOTYET
```

Comment:
For the actual PR, please remove this bit from here and simply add a comment to where it ends up that you'd like to move it back.